### ISSUE TITLE
Input does not appear when using Webpack 2 and postcss

### ISSUE THREAD

**Issue by mikeyGlitz** [react-toolbox/react-toolbox#1540]
I'm having problems showing inputs in my views:
<img width="1350" alt="screen shot 2017-06-16 at 4 36 42 pm" src="https://user-images.githubusercontent.com/606924/27243999-9dbb8754-52b2-11e7-8382-83303c5af7b5.png">
This is what the elements look like in the React Developer Tools
<img width="1440" alt="screen shot 2017-06-16 at 4 37 49 pm" src="https://user-images.githubusercontent.com/606924/27244023-b09bb2cc-52b2-11e7-9387-1dd325977bfe.png">
<img width="1440" alt="screen shot 2017-06-16 at 4 37 45 pm" src="https://user-images.githubusercontent.com/606924/27244024-b09ffa8a-52b2-11e7-8e7f-c6bd95353021.png">

Here's the code I used:
__webpack.config.js__
```js
const baseConfig = {
  entry: {
    app: main,
    react: coreModules.react
  },
  output: {
    path: outDir
  },
  resolve: {
    extensions: ['.js', '.jsx', '.json'],
    enforceExtension: false
  },
  module: {
    rules: [
      {
        test: /\.jsx?/,
        exclude: /node_modules/,
        loader: 'babel-loader'
      },
      {
        test: /\.json$/,
        loader: 'json-loader'
      },
      {
        test: /\.css$/,
        use: [
          { loader: 'style-loader' },
          {
            loader: 'css-loader',
            options: {
              sourceMap: isDevelopment,
              modules: isDevelopment,
              importLoaders: 1,
              localIdentName: '[name]__[local]___[hash:base64:5]'
            }
          },
          {
            loader: 'postcss-loader',
            options: {
              sourceMap: isDevelopment,
              sourceComments: isDevelopment,
              /* eslint-disable global-require */
              plugins: [
                require('postcss-cssnext')()
              ]
              /* eslint-enable */
            }
          }
        ]
      }
    ]
  },
  plugins: [
    new LodashModuleReplacementPlugin()
  ]
};
```

__app.js__
```js
import React from 'react';
import { Layout, Panel } from 'react-toolbox';
import Input from 'react-toolbox/lib/input';

export default function App() {
  return (
    <Layout>
      <Panel>
        <Input name="test" type="text" label="Test Input" icon="email" />
      </Panel>
    </Layout>
  );
}
```

I can get buttons and other `react-toolkit` components. Is there something I'm doing wrong?

**Comment by mikeyGlitz** [comment]
More information:
This seems to happen when I put the components inside of a switch statement:
```js
function ControlGroup({ input, meta: { touched, error }, choices, ...props }) {
  switch (props.type) {
    case 'dropdown':
      return (<Dropdown
        {...input}
        {...props}
      />);
      break;
    case 'checkbox':
      return (<Checkbox
        {...input}
        {...props}
      />);
      break;
    default:
      return (<Input
        {...input}
        error={touched && error}
        {...props}
      />);
      break;
  }
}
```

Without the switch, the component renders just fine

**Comment by tobiasandersen** [comment]
How are you using `ControlGroup`? And what props are you sending to it?

**Comment by mikeyGlitz** [comment]
I'm using it with `redux-form` which provides a `Field` component which wraps the input
```js
<Field name="myname" type="text" component={ControlGroup} />
...
function ControlGroup({ input, meta: { touched, error }, choices, ...props }) {
  switch (props.type) {
    case 'dropdown':
      return (<Dropdown
        {...input}
        source={choices}
        {...props}
      />);
      break;
    case 'checkbox':
      return (<Checkbox
        {...input}
        {...props}
      />);
      break;
    default:
      return (<Input
        {...input}
        error={touched && error}
        {...props}
      />);
      break;
  }
}

ControlGroup.propTypes = {
// Input provided by redux-form
input: PropTypes.object.isRequired,
type: PropTypes.string.isRequired,
label: PropTypes.string.isRequired,
// Meta provided by redux-form
meta: PropTypes.shape({
touched: PropTypes.bool,
error: PropTypes.string
}).isRequired,
choices: PropTypes.arrayOf(PropTypes.shape({
label: PropTypes.string.isRequired,
value: PropTypes.any.isRequired
})
};
```

**Comment by tobiasandersen** [comment]
That's weird, I can get it to render (no styling applied here though): https://codesandbox.io/s/8RQ3MJGl. Look at the "First Name" field.

**Comment by mikeyGlitz** [comment]
Looking at your example, something isn't looking right on my end:
![image](https://user-images.githubusercontent.com/606924/27388191-cda0882a-5668-11e7-8beb-be165331aedf.png)

**Comment by tobiasandersen** [comment]
Yeah there's no styling. Was that your problem as well? I thought your components didn't even render...

**Comment by mikeyGlitz** [comment]
Based on the DOM explorer, the components were present; however, I think the height and width were set to 0. When I changed the properties using the developer tools, the result looks weird.

**Comment by tobiasandersen** [comment]
I just tried it locally with webpack setup properly, with that exact same code, and it seems to work fine.

**Comment by mikeyGlitz** [comment]
I'll look at my configuration again. Maybe there's something I did wrong. Could you share your webpack config?

**Comment by mikeyGlitz** [comment]
After some experimenting, I think the problem had something to do with the fact that I was using pre-made configurations from `@easy-webpack`. I've gone through a few tutorials and referenced the [react-toolbox example](https://github.com/react-toolbox/react-toolbox-example) and got it to work. I've updated a skeleton project that I've been maintaining in the past and the configuration that I made can be found [there](https://github.com/mikeyGlitz/react-esnext-webpack-skeleton).
